virt/live_migration/virt-about-live-migration.adoc

```diff
@@ -0,0 +1,48 @@
+
+toc::[]
+
+Live migration is the process of moving a running virtual machine (VM) to another node in the cluster without interrupting the virtual workload.
+
+By default, live migration traffic is encrypted using Transport Layer Security (TLS).
+
+You can xref:../../virt/live_migration/virt-initiating-live-migration.adoc#virt-initiating-live-migration[initiate live migration manually] for a specific VM.
+
+You can monitor the progress of all live migrations on the xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-migrations_virt-web-console-overview[*Overview* -> *Migrations* tab] in the web console.
+
+You can view the metrics of a VM migration on the xref:../../virt/getting_started/virt-web-console-overview.adoc#virtualmachine-details-metrics_virt-web-console-overview[*VirtualMachine details* -> *Metrics* tab] in the web console.
+
+.Prerequisites
+
+* The cluster must have shared storage with `ReadWriteMany` (RWX) access mode.
+* The cluster must have sufficient RAM and network bandwidth.
++
+[NOTE]
+====
+You must ensure that there is enough memory request capacity in the cluster to support node drains that result in live migrations. You can determine the approximate required spare memory by using the following calculation:
+
+----
+Product of (Maximum number of nodes that can drain in parallel) and (Highest total VM memory request allocations across nodes)
+----
+
+The default xref:../../virt/live_migration/virt-configuring-live-migration#virt-configuring-live-migration-limits_virt-configuring-live-migration[number of migrations that can run in parallel] in the cluster is 5.
+====
++
+You can configure live migration limits and timeouts by using the xref:../../virt/live_migration/virt-configuring-live-migration.adoc#virt-configuring-live-migration-limits_virt-configuring-live-migration[command line].
++
+You can configure the maximum number of migrations per node or cluster by using the xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-settings-cluster_virt-web-console-overview[web console].
+
+* If a VM uses a host model CPU, the nodes must support the CPU.
+* A xref:../../virt/vm_networking/virt-dedicated-network-live-migration.adoc#virt-dedicated-network-live-migration[dedicated Multus network] for live migration is highly recommended. A dedicated network minimizes the effects of network saturation on tenant workloads during migration.
++
+You can select a dedicated live migration network by using the xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-settings-cluster_virt-web-console-overview[web console].
+
+[role="_additional-resources"]
+[id="additional-resources_virt-live-migration"]
+== Additional resources
+* xref:../../virt/monitoring/virt-prometheus-queries.adoc#virt-live-migration-metrics_virt-prometheus-queries[Prometheus queries for live migration]
+* link:https://access.redhat.com/articles/6994974#vm-migration-tuning[VM migration tuning]
```

Comment:
I may have gone overboard here, but I have a suggestion for reworking this assembly. 😅 
```suggestion
:_content-type: ASSEMBLY
[id="virt-about-live-migration"]
= About live migration
include::_attributes/common-attributes.adoc[]
:context: virt-about-live-migration

toc::[]

Live migration is the process of moving a running virtual machine (VM) to another node in the cluster without interrupting the virtual workload. By default, live migration traffic is encrypted using Transport Layer Security (TLS).

[id="requirements_virt-about-live-migration"]
== Requirements

* The cluster must have shared storage with `ReadWriteMany` (RWX) access mode.
* The cluster must have sufficient RAM and network bandwidth.
+
[NOTE]
====
You must ensure that there is enough memory request capacity in the cluster to support node drains that result in live migrations. You can determine the approximate required spare memory by using the following calculation:

----
Product of (Maximum number of nodes that can drain in parallel) and (Highest total VM memory request allocations across nodes)
----

The default xref:../../virt/live_migration/virt-configuring-live-migration#virt-configuring-live-migration-limits_virt-configuring-live-migration[number of migrations that can run in parallel] in the cluster is 5.
====

* If a VM uses a host model CPU, the nodes must support the CPU.
* A xref:../../virt/vm_networking/virt-dedicated-network-live-migration.adoc#virt-dedicated-network-live-migration[dedicated Multus network] for live migration is highly recommended. A dedicated network minimizes the effects of network saturation on tenant workloads during migration.

[id="common-live-migration-tasks_virt-about-live-migration"]
== Common live migration tasks

* You can xref:../../virt/live_migration/virt-initiating-live-migration.adoc#virt-initiating-live-migration[initiate live migration manually] for a specific VM.

* You can xref:../../virt/live_migration/virt-configuring-live-migration.adoc#virt-configuring-live-migration-limits_virt-configuring-live-migration[configure live migration limits and timeouts] by using the command line.

* You can configure a xref:../../virt/vm_networking/virt-dedicated-network-live-migration.adoc#virt-dedicated-network-live-migration[dedicated Multus network] for live migration.

[id="web-console-tasks_virt-about-live-migration"]
=== Web console tasks

* You can select a dedicated live migration network on the xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-settings-cluster_virt-web-console-overview[*Settings* -> *Cluster* tab] in the web console.

* You can monitor the progress of all live migrations on the xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-migrations_virt-web-console-overview[*Overview* -> *Migrations* tab] in the web console.

* You can view the metrics of a VM migration on the xref:../../virt/getting_started/virt-web-console-overview.adoc#virtualmachine-details-metrics_virt-web-console-overview[*VirtualMachine details* -> *Metrics* tab] in the web console.

* You can configure the maximum number of migrations per node or cluster on the xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-settings-cluster_virt-web-console-overview[*Settings* -> *Cluster* tab] in the web console.

[role="_additional-resources"]
[id="additional-resources_virt-live-migration"]
== Additional resources
* xref:../../virt/monitoring/virt-prometheus-queries.adoc#virt-live-migration-metrics_virt-prometheus-queries[Prometheus queries for live migration]
* link:https://access.redhat.com/articles/6994974#vm-migration-tuning[VM migration tuning]
```

Reply:
I like the idea of grouping tasks. The only thing I don't like is "web console tasks", because the user will look for a task according to function, not according to the tool that they need to use. I will definitely try this!